syntax = "proto3";
message User {
string id = 1;
string name = 2;
string email = 3;
}
message Post {
string id = 1;
string title = 2;
string content = 3;
repeated string tags = 4;
}
message GetUserRequest {
string user_id = 1;
}
message GetUserResponse {
User user = 1;
}
service UserService {
rpc GetUser (GetUserRequest) returns (GetUserResponse) {}
}